libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -370,6 +395,19 @@ public void addRemotePublisher(Class<? extends Transform> transformer,
         remotes.put(transformer, publisher);
     }
 
+    /**
+     * Removes a remote publisher to publish events to.
+     *
+     * @param transformer Transformer class.
+     */
+    public void removeRemotePublisher(Class<? extends Transform> transformer) {
```

Comment:
Why do we need this? I'd prefer not adding this to our public API.

Reply:
I can limit visibility. It's required because by default new instances `SalesforceAnalyticsManager` have the AILTN publisher added to them. In order to test this class the AILTN publisher needs to be removed.